packaging/os/pkgng.py

```diff
@@ -24,10 +24,10 @@
 DOCUMENTATION = '''
 ---
 module: pkgng
-short_description: Package manager for FreeBSD >= 9.0
+short_description: Package manager for FreeBSD >= 8.3 and >= 9.1
```

Comment:
I am not exactly sure, but the new package manager is part of the official release since version 9.1. It might be possible that there was an option to use pkgng with the 8.3 release as well.

But I would prefer to stick to the offical [man page](https://www.freebsd.org/cgi/man.cgi?query=pkg&apropos=0&sektion=0&manpath=FreeBSD+10.3-RELEASE+and+Ports&arch=default&format=html) which states:

> The pkg command first appeared in FreeBSD 9.1.